app/controllers/admin/items_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::ItemsController < Admin::BaseController
+  def new
+    @item = Item.new
+    @categories = Category.all
+  end
+
+  def create
+  @item = Item.create(item_params)
+    if @item.save
+      flash[:notice] = "#{@item.title} has been created!"
+      redirect_to @item
+    else
+      flash.now[:errors] = @item.errors.full_messages.join(', ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.